### PR TITLE
Add login page

### PR DESCRIPTION
**Pull request by geekingreen**
This is based on [LoginPage_1.pdf](https://github.com/VillageBookBuilders/vbb-portal-web/files/8027078/LoginPage_1.pdf) design

image
image

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <Typography
+          variant="h6"
+          component="div"
+          align="center"
+          sx={flexContainerStyles}
+        >
+          Village Portal
+        </Typography>
+        <Box display="flex" justifyContent="flex-end" sx={flexContainerStyles}>
+          <Button color="inherit" component={Link} to="/login" sx={{ mr: 2 }}>
```
Comment (wley3337) [inline src/components/Header.tsx:32]: Non-blocking: do we want to start a theme yet? Or do the theme in a single PR?

File: src/App.tsx [hunk 2]
```diff
@@ -1,32 +1,19 @@
...
-          Login
-        </Button>
-      </Toolbar>
-    </AppBar>
+    <Header />
     <Box pt="64px">
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/login" element={<Login />} />
+        <Route path="/register" element={<Register />} />
```
Comment (wley3337) [inline src/App.tsx:16]: ? not sure if we want to establish a pattern of using `exact` unless we need to have something show up at a top level of a route 🤷 ( thinking of later folks who might pattern match and not quite understand why something is still showing up )

File: src/components/Header.tsx (hunk 1 above)
Comment (geekingreen) [reply src/components/Header.tsx:32]: I've created a theme but haven't applied any styles as I'm not sure what we want, but it's ready to go should we decide on something. But I can totally throw something together in this or another PR if we have an idea of what we want: https://github.com/VillageBookBuilders/vbb-portal-web/blob/main/src/theme.ts

File: src/App.tsx (hunk 2 above)
Comment (geekingreen) [reply src/App.tsx:16]: We are using react-router v6 which uses route ranking and nested routing https://reactrouter.com/docs/en/v6/getting-started/concepts#ranking-routes So I think the exact paradigm doesn't apply anymore (if it's even an option on the Route component)

**Review by wley3337 — APPROVED** [review]

File: src/App.tsx (hunk 2 above)
Comment (wley3337) [reply src/App.tsx:16]: That's supper cool! I'll need you to keep an eye on me so I don't fall back into an old pattern 😁 . I really like the nesting routes pattern too!

**Pull request merged**